storm/Error.hpp

```diff
@@ -4,10 +4,14 @@
 #include <cstdint>
 
 #if defined(WHOA_SYSTEM_WIN)
-#include <winerror.h>
+// This is necessary to prevent issues
+// with std::max() or std::min() when including this header file
+#define NOMINMAX
+#include <Windows.h>
```

Comment:
I had to include the whole shazam, because of SetLastError ()

Reply:
Two quick things here: let's move `#include <windows.h>` into `storm/Error.cpp`, and #19 kicks on both `NOMINMAX` and `WIN32_LEAN_AND_MEAN` for all of Squall, so we can drop the one-off `#define` here.